tests/test-sources/plugins/none-ls.nix

```diff
@@ -100,95 +99,70 @@
-              # As of 2024-05-22, python311Packages.k5test (one of ansible-lint's dependenvies) is broken on darwin
-              # TODO: re-enable this test when fixed
-              "ansible_lint"
-              "clazy"
-              "gdformat"
-              "gdlint"
-              "haml_lint"
-              # As of 2024-06-29, pkgs.rubyfmt is broken on darwin
-              # TODO: re-enable this test when fixed
-              "rubyfmt"
-              "verilator"
-              "verible_verilog_format"
-            ])
-            ++ (pkgs.lib.optionals pkgs.stdenv.isAarch64 [
-              "semgrep"
-              "smlfmt"
-              # As of 2024-03-11, swift-format is broken on aarch64
-              # TODO: re-enable this test when fixed
-              "swift_format"
-            ]);
+    module =
+      {
+        config,
+        options,
+        lib,
+        pkgs,
+        ...
+      }:
+      {
+        plugins.none-ls = {
+          # sandbox-exec: pattern serialization length 159032 exceeds maximum (65535)
+          enable = !pkgs.stdenv.isDarwin;
 
-          sources = pkgs.lib.mapAttrs (
-            _: sources:
-            pkgs.lib.mapAttrs (
-              source: _:
-              { enable = true; } // pkgs.lib.optionalAttrs (builtins.elem source unpackaged) { package = null; }
-            ) sources
-          ) options;
-        in
-        sources;
-    };
+          sources =
+            let
+              disabled =
+                [
+                  # As of 2024-03-22, pkgs.d2 is broken
+                  # TODO: re-enable this test when fixed
+                  "d2_fmt"
+                  # TODO: can this be re-enabled?
+                  "yamlfix"
+                ]
+                ++ (lib.optionals (pkgs.stdenv.isDarwin && pkgs.stdenv.isx86_64) [
+                  # As of 2024-03-27, pkgs.graalvm-ce (a dependency of pkgs.clj-kondo) is broken on x86_64-darwin
+                  # TODO: re-enable this test when fixed
+                  "clj_kondo"
+                ])
+                ++ (lib.optionals pkgs.stdenv.isDarwin [
+                  # As of 2024-05-22, python311Packages.k5test (one of ansible-lint's dependenvies) is broken on darwin
+                  # TODO: re-enable this test when fixed
+                  "ansible_lint"
+                  "clazy"
+                  "gdformat"
+                  "gdlint"
+                  "haml_lint"
+                  # As of 2024-06-29, pkgs.rubyfmt is broken on darwin
+                  # TODO: re-enable this test when fixed
+                  "rubyfmt"
+                  "verilator"
+                  "verible_verilog_format"
+                ])
+                ++ (lib.optionals pkgs.stdenv.isAarch64 [
+                  "semgrep"
+                  "smlfmt"
+                  # As of 2024-03-11, swift-format is broken on aarch64
+                  # TODO: re-enable this test when fixed
+                  "swift_format"
+                ]);
```

Comment:
Have you checked that all the packages marked as broken are still broken ?

Reply:
No I haven't. Is the best way to just empty the list and see what buildbot does?

Reply:
Yeah I think... We can punt that to another PR